### DISCUSSION TITLE
Flutter: How to get ride of "Prefer const with const constructor"

### DISCUSSION THREAD

**Discussion by amanshaya12** [community/community#21304] [Other Feature Feedback, Questions, & Ideas] [answered]
I often have to write manually "const" keyword to widget,
I searching for any alternative there can automatically add const to const

**Comment by surajadkhari** [comment, marked as answer]
1.Ctrl+Shift+P-->Preference Open Setting(JSON)
2. Paste the following
"editor.codeActionsOnSave": {     "source.fixAll": true }

For your reference read this article:
https://towardsdev.com/flutter-linter-warnings-due-to-missing-const-constructors-82db12c37113

**Reply by amanshaya12** [reply]
Thank you so much suraj
